database/migrations/2017_04_26_133415_create_trader_user_pivot_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTraderUserPivotTable extends Migration
```

Comment:
Could this many to many be simplified if we had Customers and Traders - both extended User?